packages/email/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "Send email messages",
-  version: "1.1.15-beta.10"
+  version: "1.1.16-beta.11"
```

Comment:
When running `meteor update --release 1.4-beta.11` I see:

```
=> Errors while initializing project:

While checking for email@1.1.15-beta.11:
error: No compatible binary build found for this package. Contact the package author and ask them
to publish it for your platform.
```

explicitly adding `email@1.1.16-beta.11` to my .meteor/packages gets me through, but I figured it would help to note.  Platform OSX if that matters!